docker/test/util/process_functional_tests_result.py

```diff
@@ -12,7 +12,7 @@
 SKIPPED_SIGN = "[ SKIPPED "
 HUNG_SIGN = "Found hung queries in processlist"
 
-NO_TASK_TIMEOUT_SIGNS = ["All tests have finished", "No tests were run"]
+SUCCESS_FINISH_SIGNS = ["All tests have finished", "No tests were run"]
```

Comment:
Btw, usually `No tests were run` is not a success sign. As far as I remember, we have added this check for `All tests have finished` because we had test statuses like 'passed: 0, failed: 0, skipped: 0' with green tick when tests runner was actually broken (and it's hard to notice that something's broken when all checks are green).

Reply:
Should I replace it by just `TASKS_FINISHED_SIGNS`?